source/boundary_traction/interface.cc

```diff
@@ -60,9 +60,22 @@ namespace aspect
     Interface<dim>::parse_parameters (dealii::ParameterHandler &)
     {}
 
+    template <int dim>
```

Comment:
We use three empty lines between functions in .cc files (one empty line in .h files):
```suggestion


    template <int dim>
```